contrib/start_development_backend

```diff
@@ -8,19 +8,20 @@ function _print_syntax() {
   echo "Usage: $me [-d <dir> -l <dir> -w <num> -r <server>]"
   echo -e "\t-d <dir>\tDirectory of source code. If not provided /vagrant ist used"
   echo -e "\t-l <dir>\tDirectory where the logfiles should be put. If not set STDERR and STDOUT will be printed to console"
-  echo -e "\t-r <server>\tRegister to this reposerver. Can be used multiple times."
+  echo -e "\t-h <string>\tSet hostname of server. (default: backend)"
```

Comment:
💭 this is because you need to set this to `localhost` right?

Reply:
Yes. Or whatever hostname you like :)